### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by matthewdavidson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,23 +17,27 @@
...
+    "jest": "^22.4.0",
+    "stylelint": "^9.1.1"
   },
   "peerDependencies": {
-    "stylelint": "^8.0.0"
+    "stylelint": "^9.1.1"
   },
   "dependencies": {
-    "stylelint-config-standard": "17.0.0",
+    "postcss-less": "^1.1.3",
```
Comment (danrr) [inline package.json]: `postcss-less`?

Comment (matthewdavidson) [reply package.json]: I was fixing these warnings:
```
npm WARN postcss-html@0.12.0 requires a peer of postcss-less@>=1.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-html@0.12.0 requires a peer of postcss-scss@>=1.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-html@0.12.0 requires a peer of sugarss@>=1.0.0 but none is installed. You must install peer dependencies yourself.
```

Comment (danrr) [reply package.json]: It looks like all of these should come with stylelint: https://github.com/stylelint/stylelint/blob/master/package.json#L63

Is that error caused by a conflict then?

**Comment by matthewdavidson** [comment]
@danrr fixed. Also should a change in peerDependency (major bump) result in major for this package?

**Comment by danrr** [comment]
Make it major, with a patch bump in backpack-react-scripts.

**Comment by matthewdavidson** [comment]
@danrr i need approval 🙏  (on this PR mostly)

**Comment by danrr** [comment]
@matthewdavidson You are a nice person! Also, you can merge this. Does that help?

**Pull request merged**